### ISSUE TITLE
altitude option in  featureToPolygon #Feature2Mesh

### ISSUE THREAD

**Issue by Oncledjoe** [iTowns/itowns#997]
Hello, 
This issue is about the projection of a polygon, which came from a wfs stream, with no Z, on the elevation layer. As we can see in example/globe_wfs_extruded.html it works for LineString( `function altitudeLine(properties, contour)`). 
The idea is to implement a similar function for polygon. But in Feature2Mesh if we follow the same process than linestring it doesn't work. 
It is the fault of getProperty called in featureToPolygon, it always returns 0 (it is because it tries to execute the function: property(...args) equivalent to options.altitude(geometry.properties) when i need options.altitude(feature.vertices)). 
Furthermore getProperty is not used in featureToLine for the same operation, hence its success.
In resume: Can you fix featureToPolygon to make it look like at featureToLine with sames functionality. 
I hope to be clear. Thanks a lot.
Best Regards, 
Oncledjoe

**Comment by gchoqueux** [comment]
It's a regularly requested feature. It would be great to clamp object to terrain with `feature2Mesh`.

**Comment by Oncledjoe** [comment]
I agree, I develop a scientist web platform for cnrs and most of spatial data are polygon 2D. Without this feature, Itwons becomes less interesting. A quick modification of featureToPolygon give that: 
![polygononelevationlayer](https://user-images.githubusercontent.com/36001626/51331816-d3028200-1a7a-11e9-9b6a-d687f4ce24ac.png)
It wills not work for very sharply terrain I thinks, but it's a beginning.

**Comment by gchoqueux** [comment]
if you won't project 2D polygons from WFS on the terrain you could use `ColorLayer`, see [this example](http://www.itowns-project.org/itowns/examples/globe_wfs_color.html)
